### PR TITLE
Chapter 7 inheritance

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/inheritance/reading/cat-classes/_index.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+---
+title: "A Tale of Two Cats"
+date: 2021-10-01T09:28:27-05:00
+weight: 3
+draft: false
+---
+
+Let’s revisit our `Cat` and `HouseCat` friends. In `java-web-dev-exercises`, open `src/org/launchcode/java/demos/inheritance` and examine the two classes inside.
```
Comment (codinglikeagirl42) [inline content/inheritance/reading/cat-classes/_index.md:15]:
```suggestion
Let’s revisit our `Cat` and `HouseCat` friends. In `java-web-dev-project-17`, open `src/main/java/org.launchcode.java.demos/lsn6inheritance` and examine the two classes inside.
```

File: content/inheritance/reading/cat-classes/_index.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+## `Object` Class
+
+In a previous lesson, we introduced the special methods `equals` and `toString`. All classes contain default implementations of these methods that can be overridden. In fact, these default methods are part of a class called `Object`. If a class does not explicitly extend another class, then it implicitly extends `Object`. So the default implementations of `equals` and `toString` (along with a few other methods) are made available to us via inheritance.
+
+Note that we should use the `@Override` annotation when we provide new implementations of these methods as well.
+
+### References:
+- [The @Override annotation (docs.oracle.com)](https://docs.oracle.com/en/java/javase/17/docs/api/index.html)
+- [The Object Class (docs.oracle.com)](https://docs.oracle.com/en/java/javase/17/docs/api/index.html)
```
Comment (codinglikeagirl42) [inline content/inheritance/reading/cat-classes/_index.md:158]:
```suggestion
- [The Object Class (docs.oracle.com)](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html
```

Comment (codinglikeagirl42) [inline content/inheritance/reading/cat-classes/_index.md]: Please update links to be more specific.

File: content/inheritance/exercises/_index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Exercises: Inheritance"
+date: 2021-10-01T09:28:27-05:00
+weight: 2
+draft: false
+---
+
+Work on these exercises within IntelliJ using the `java-web-dev-projects` project you already created. Add a new package called `technology` to your `exercises` directory.
```
Comment (codinglikeagirl42) [inline content/inheritance/exercises/_index.md:15]:
```suggestion
Work on these exercises within IntelliJ using the `java-web-dev-projects-17` project you already created. Add a new package called `technology` to your `exercises` directory.
```

**Review by codinglikeagirl42 — APPROVED** [review]
Just a few suggestions to look over before merging.  Also noticed that the metadata is different in this chapter, please update.

File: content/inheritance/reading/cat-classes/_index.md (hunk 2 above)
Comment (jwoolbright23) [reply content/inheritance/reading/cat-classes/_index.md]: updated links

**Pull request merged**